atomic_reactor/util.py

```diff
@@ -785,3 +787,71 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
     return blob_config
+
+def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, parent_env=None):
```

Comment:
I don't follow, it only adds one option that the DockerfileParser itself does not already offer. How is that too many?